### PR TITLE
Add return type to Argument Reference on vpc_endpoint_service document

### LINKED ISSUE #15197: Exported value aws_vpc_endpoint_service of aws_vpc_endpoint_service has no indices
### Community Note
* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform CLI and Terraform AWS Provider Version
$ terraform -v
Terraform v0.13.2
+ provider registry.terraform.io/hashicorp/aws v3.6.0

### Affected Resource(s)
aws_vpc_endpoint_service, exporting single indexed value of base_endpoint_dns_names

### Terraform Configuration Files
```hcl
output "endpointinfo" {
  value = aws_vpc_endpoint_service.epsvc.base_endpoint_dns_names.0
}
```

### Debug Output
https://gist.github.com/tezzigator/42966402f94d12bc8eef573f37c595f2

### Expected Behavior

Expected to see the leading string of that list output

### Actual Behavior
```hcl
Error: Invalid index

  on modules/workload/main.tf line 202, in output "endpointinfo":
 202:   value = aws_vpc_endpoint_service.epsvc.base_endpoint_dns_names.0

This value does not have any indices.
```

### Important Factoids
If I simply export the entire aws_vpc_endpoint_service then I do get an actual list:
```hcl
orangeendpointsvc = [
  "vpce-svc-REDACTED.us-east-1.vpce.amazonaws.com",
]
```
**But that comma there concerns me - seems like it shouldnt be there; I supposed that could be causing all this?**
* #0000

### PR DESCRIPTION
**Pull request by daisakuhazui**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates #15197

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
None
```

### CODE REVIEW

File: website/docs/r/vpc_endpoint_service.html.markdown [hunk 1]
```diff
@@ -54,15 +54,15 @@ The following arguments are supported:
...
-* `availability_zones` - The Availability Zones in which the service is available.
-* `arn` - The Amazon Resource Name (ARN) of the VPC endpoint service.
-* `base_endpoint_dns_names` - The DNS names for the service.
-* `manages_vpc_endpoints` - Whether or not the service manages its VPC endpoints - `true` or `false`.
-* `private_dns_name` - The private DNS name for the service.
-* `service_name` - The service name.
-* `service_type` - The service type, `Gateway` or `Interface`.
-* `state` - The state of the VPC endpoint service.
+* `id` - string - The ID of the VPC endpoint service.
+* `availability_zones` - list(string) - The Availability Zones in which the service is available.
```
Comment (philnichol) [inline website/docs/r/vpc_endpoint_service.html.markdown:58]: Looks like availability_zones is actually set(string)

Comment (daisakuhazui) [reply website/docs/r/vpc_endpoint_service.html.markdown:58]: Ops, sorry. I would fix it.

https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/data_source_aws_vpc_endpoint_service.go#L30

**Comment by daisakuhazui** [comment]
Oops... I have removed `terraform-providers/aws-provider` from the reviewer... 😢

File: website/docs/r/vpc_endpoint_service.html.markdown [hunk 2]
```diff
@@ -54,15 +54,15 @@ The following arguments are supported:
...
-* `state` - The state of the VPC endpoint service.
+* `id` - string - The ID of the VPC endpoint service.
+* `availability_zones` - set(string) - The Availability Zones in which the service is available.
+* `arn` - string - The Amazon Resource Name (ARN) of the VPC endpoint service.
+* `base_endpoint_dns_names` - set(string) - The DNS names for the service.
+* `manages_vpc_endpoints` - bool - Whether or not the service manages its VPC endpoints - `true` or `false`.
+* `private_dns_name` - string - The private DNS name for the service.
+* `service_name` - string - The service name.
+* `service_type` - string - The service type, `Gateway` or `Interface`.
+* `state` - string - The state of the VPC endpoint service.
```
Comment (breathingdust) [inline website/docs/r/vpc_endpoint_service.html.markdown:65]: Hi @daisakuhazui! Until we have a decision on #15365, lets keep with the standard that is most consistent across the provider and just mention Set in the description.

```suggestion
* `id` - The ID of the VPC endpoint service.
* `availability_zones` - A set of Availability Zones in which the service is available.
* `arn` - The Amazon Resource Name (ARN) of the VPC endpoint service.
* `base_endpoint_dns_names` - A set of DNS names for the service.
* `manages_vpc_endpoints` -  Whether or not the service manages its VPC endpoints - `true` or `false`.
* `private_dns_name` - The private DNS name for the service.
* `service_name` - The service name.
* `service_type` - The service type, `Gateway` or `Interface`.
* `state` - The state of the VPC endpoint service.
```

**Review by breathingdust — CHANGES_REQUESTED** [review]

**Pull request merged**